ci-operator/step-registry/ibmcloud/provision/byok/ibmcloud-provision-byok-commands.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+# IBM Cloud CLI login
+function ibmcloud_login {
+    export IBMCLOUD_CLI=ibmcloud
+    export IBMCLOUD_HOME=/output   
+    region="${LEASED_RESOURCE}"
+    export region
+    echo "Try to login..." 
```

Comment:
Insert `"${IBMCLOUD_CLI}" config --check-version=false` before login, to avoid ibmcloud plugin check.

Reply:
I already did that in https://github.com/openshift/release/pull/46776